README.md

```diff
@@ -0,0 +1,35 @@
+# Phel Lang Support for VS Code
+
+This VS Code plugin gives very basic support for the Phel Language. Currently only syntax highlighting is supported.
+
+## Install
+
+This plugin is currently __not__ installable via the VS Code marketplace. To install the plugin you must manually copy or link the code of this plugin to the extension directory. Depending on your platform, the location is in the following folder:
+
+* Windows `%USERPROFILE%\.vscode\extensions`
+* macOS `~/.vscode/extensions`
+* Linux `~/.vscode/extensions`
+
+### Example installation on Linux:
```

Comment:
@jenshaase also available on Mac, isn't? 👀

Reply:
Yes, I should be possible to use it on a Mac. But I don't have a Mac. So I not sure how to install it correctly.

Reply:
I think @chemaclass already installed the plugin on his Mac.
Can you update the file? :-)